py/README.rst

```diff
@@ -8,6 +8,7 @@ to use this package for scoring with Driverless AI MOJO models.
 
 PySparkling Documentation is hosted at our documentation page:
 
+- For Spark 3.5 - http://docs.h2o.ai/sparkling-water/3.5/latest-stable/doc/pysparkling.html
```

Comment:
Shouldn't the changes in this file be rather a part of https://github.com/h2oai/sparkling-water/pull/5664?